catroid/src/org/catrobat/catroid/content/Project.java

```diff
@@ -424,4 +421,33 @@ public boolean containsSprite(Sprite selectedSprite) {
 		}
 		return false;
 	}
+
+	public void addSprite(int spriteIndex, Sprite sprite) {
+		spriteList.add(spriteIndex, sprite);
+	}
+
+	public void refreshSpriteReferences() {
+		Project project = ProjectManager.getInstance().getCurrentProject();
+		for (Brick brick : project.getAllBricks()) {
+			if (!(brick instanceof PointToBrick)) {
+				continue;
+			}
+			PointToBrick pointToBrick = (PointToBrick) brick;
+			Sprite pointedSprite = pointToBrick.getPointedObject();
+			if (pointedSprite == null) {
+				continue;
+			}
+
+			Sprite newSprite = getSpriteBySpriteName(pointToBrick.getPointedObject());
```

Comment:
Yes, thx for the hint! Should be fixed now (I've added a check in PointToBrick.java)! ;)